content/js2/blocks/query-string/index.md

```diff
@@ -7,18 +7,18 @@ emoji= '🧩'
 [objectives]
 1='Identify the query string section of a url'
 2='Identify query parameters within a query string'
-3='Explain why an object is a more useful structure for storing query parameters'
+3='Explain why an object is a more useful structure for storing query parameters than'
 +++
 
 Let’s define a problem.
 
-Websites have addresses called urls like this: "https://example.com". Urls often have {{<tooltip title="query strings">}}Query strings go at the end of a url and are used to specify more information about the content you get back from a request to a server{{</tooltip>}}too. Here is an example of a url with a query string on the end:
+Websites have addresses called urls like this: "https://example.com/widgets". Urls often have {{<tooltip title="query strings">}}Query strings go at the end of a url and are used to specify more information about the content you get back from a request to a server{{</tooltip>}}too. Here is an example of a url with a query string on the end:
 
 https://example.com/widgets?colour=blue&sort=newest
 
 For the url above, the **query string** is `"colour=blue&sort=newest"`. Query strings consist of **query parameters**, separated by an ampersand character `&`. `colour=blue` is a query parameter: we say that `colour` is the key and `blue` is the value.
 
-urls must always be strings. However, a string isn't a useful data type for accessing query parameters. Given a key, accessing the value from a query string is not straightforward However, objects are ideal for looking up values with keys.
+urls must always be strings. However, a string isn't a useful data type for accessing query parameters. Given a key like `colour`, accessing the value from a query string stored as a string is not straightforward. However, objects are ideal for looking up values with keys.
```

Comment:
Accessing the value from a query string stored as a string doesn't scan well - can we leave it as query string?

Reply:
Works for me :)